src/utils/util/resolver-pipeline/select-or-custom-merge.xsl

```diff
@@ -11,21 +11,22 @@
 
     <xsl:include href="oscal-profile-resolve-functions.xsl"/>
 
-    <!-- A control is included if it is selected by the provided import instruction -->
-    <xsl:template match="control" mode="o:select" as="element(o:control)?">
+    <!-- A control is included if it is selected by the provided import instruction. -->
+    <!-- Also, for a nonselected control, this template can return selected children. -->
+    <xsl:template match="control" mode="o:select" as="element(o:control)*">
```

Comment:
Line 16 has the bug fix. Everything else in this pull request is testing the fix or opportunistically fixing indentation.